volumes/miovision/sql/create-table-classifications.sql

```diff
@@ -1,11 +1,11 @@
 ﻿CREATE TABLE miovision_api.classifications
 (
-    classification_uid integer NOT NULL DEFAULT nextval('miovision_api.classifications_classification_uid_seq'::regclass),
+    classification_uid integer DEFAULT nextval('miovision_api.classifications_classification_uid_seq'::regclass) PRIMARY KEY,
     classification text COLLATE pg_catalog."default",
     location_only boolean, -- for peds and bikes, where movement isn't available, only which leg they were observed on
     class_type text COLLATE pg_catalog."default"
 )
 WITH (
     OIDS = FALSE
 )
-TABLESPACE pg_default;
\ No newline at end of file
+TABLESPACE pg_default;
```

Comment:
can we add a  `Alter table ... owner to miovision_admins; ` ? Realized it was missing from the original file.

Reply:
Done!